src/test/java/net/sf/jabref/logic/openoffice/OOBibStyleTest.java

```diff
@@ -494,4 +494,26 @@ public void testCompareToNotEqual() throws IOException {
         assertTrue(style1.compareTo(style2) > 0);
         assertFalse(style2.compareTo(style1) > 0);
     }
+
+
+    public void test1245AndOxfordComma() throws URISyntaxException, IOException {
```

Comment:
Something with empty strings as values?! maybe the oxford comma is even a separate test case?